source/_patterns/01-atoms/button/index.js

```diff
@@ -0,0 +1,12 @@
+// import $ from 'jquery';
+// might be good to force in some jquery here
+import button from 'bootstrap/js/src/button';
```

Comment:
Verifying: this works?

Reply:
Yep, this works. The main thing I was looking for here was that "toggled" buttons stay toggled (http://getbootstrap.com/docs/4.0/components/buttons/#toggle-states)